recipe_scrapers/theglutenfreeaustrian.py

```diff
@@ -8,9 +8,17 @@ def host(cls):
         return "theglutenfreeaustrian.com"
 
     def ingredient_groups(self):
-        return group_ingredients(
+        ingredient_groups = group_ingredients(
             self.ingredients(),
             self.soup,
             ".mv-create-ingredients h4",
             ".mv-create-ingredients ul li",
         )
+
+        schema_title = self.schema.title()
+
+        for group in ingredient_groups:
+            if group.purpose and schema_title in group.purpose:
```

Comment:
I'd probably suggest using an equality-check here; what do you reckon?

```suggestion
            if group.purpose == schema_title:
```